pliers/extractors/audio.py

```diff
@@ -155,27 +155,74 @@ def get_feature_names(self):
         return self._feature
 
     def _get_values(self, stim):
-        if self._feature in ['rmse', 'zero_crossing_rate']:
+        if self._feature in ['rmse', 'zero_crossing_rate', 'rms', 'spectral_flatness']:
```

Comment:
See @qmac's comment—we should drop `'rmse'` in favor of `'rms'`, and pin the librosa version number when this changed in our requirements.